great_expectations/experimental/datasources/interfaces.py

```diff
@@ -47,26 +48,22 @@
     pyspark_sql_Row = None
 
 # BatchRequestOptions is a dict that is composed into a BatchRequest that specifies the
-# Batches one wants returned. The keys represent dimensions one can slice the data along
+# Batches one wants as returned. The keys represent dimensions one can slice the data along
 # and the values are the realized. If a value is None or unspecified, the batch_request
 # will capture all data along this dimension. For example, if we have a year and month
-# splitter and we want to query all months in the year 2020, the batch request options
+# splitter, and we want to query all months in the year 2020, the batch request options
 # would look like:
 #   options = { "year": 2020 }
 BatchRequestOptions: TypeAlias = Dict[str, Any]
 
 
-@dataclasses.dataclass(frozen=True)
+@pydantic_dc.dataclass(frozen=True)
```

Comment:
This question is for the whole team.
Do we want to do this?
It's not strictly necessary because all of the experimental core is under type-checking coverage.
And we don't need to `serde` it.

Reply:
@Kilo59 Should I revert it?